ui/src/pages/insightDetail/components/eventAggregator/index.tsx

```diff
@@ -67,6 +67,7 @@
   const eventSource = useRef<EventSource>()
   const diagnosisEndRef = useRef<HTMLDivElement>(null)
   const abortControllerRef = useRef<AbortController | null>(null)
+  const contentRef = useRef<HTMLDivElement>(null)
```

Comment:
The addition of `contentRef` is unclear. Could you please explain why this ref is being added and how it will be used in the component?